src/components/views/rooms/SendMessageComposer.tsx

```diff
@@ -600,7 +601,7 @@ export class SendMessageComposer extends React.Component<ISendMessageComposerPro
     };
 
     render() {
-        const threadId = this.props.relation?.rel_type === RelationType.Thread
+        const threadId = this.props.relation?.rel_type === THREAD_RELATION_TYPE.name
```

Comment:
Shouldn't this use `THREAD_RELATION_TYPE.matches(relation?.rel_type)` to support both legacy and new?